programs/hyperplane/src/utils/macros.rs

```diff
@@ -41,7 +41,7 @@ macro_rules! curve {
 macro_rules! require_msg {
     ($invariant:expr, $error:expr $(,)?, $message: expr) => {
         if !($invariant) {
-            msg!($message);
+            anchor_lang::prelude::msg!($message);
```

Comment:
If you want the "right" way to use an external crate feature in a macro:
```suggestion
            ::anchor_lang::prelude::msg!($message);
```

Reply:
Neat thanks!